_data/students.yml

```diff
@@ -0,0 +1,24 @@
+# Please add your entries to a random position in the file.
+# This is to remove conflicts with other people adding entries.
+
+- name: Miguel Piedrafita
+  github: m1guelpf
+  image: m1guelpf.png
+  blog: https://miguelpiedrafita.com/
+  twitter: m1guelpf
+
+- name: Chahat Bansal
+  github: chahatbansal
+  image: chahat.png
+
+- name: Nitesh Kumar Niranjan
+  github: niteshkumarniranjan
+  image: niteshkumarniranjan.jpg
+
+- name: Aleksandr Lymarenko
+  github: a-v-lymarenko
+  image: a-v-lymarenko.png
+
+- name: Abishek V Ashok
+  github: abishekvashok
+  image: abishek.png
```

Comment:
Where are the Other Student's Details?

Reply:
@naveentherocker I migrated them from the old system, so they aren't available. Maybe the students can PR the other details when this is merged :smile:

Reply:
@m1guelpf I think he meant other studetns. New students are adding their info on a speedy basis